packages/redux/src/sizePreferences/selectors.ts

```diff
@@ -0,0 +1,32 @@
+import { getError, getIsLoading, getResult } from './reducer.js';
+import type { SizePreferencesState } from './types/state.types.js';
+import type { StoreState } from '../types/index.js';
+
+/**
+ * Returns the sizePreferences error.
+ *
+ * @param state - Application state.
+ *
+ * @returns Error.
+ */
+export const getSizePreferencesError = (state: StoreState) =>
```

Comment:
The action is called `fetchListingFacets` but the selector does not match that name. Either all clients, actions and selectors follow the naming listingFacets or sizePreferences. In my opinion, sizePreferences is a much better name but check if that is possible. If you do choose to go with the sizePreferences, then please change every name that still refers to listingFacets to use that name (like fetchSizePreferences, getSizePreferences, etc...)

Reply:
I can see that using those two names might be confusing and I will change to one of those two. Choosing listingFacets might be the most accurate since the endpoint is retrieving listing facets, what do you think?

Reply:
I am not aware of what is the intended usage of this endpoint but if the idea is to give users a way to find the size preferences, then I would not call it listingfacets even though the endpoint is called like that. The name of our clients can be different than the name of the resources depending if we want to add an abstraction over them which will facilitate the consumers understanding. Talk to your team members about this as the name will depend on what feature are we intended to give to users with this change.